Cargo.toml

```diff
@@ -872,6 +872,17 @@ description = "Shows how to create a 3D orthographic view (for isometric-look in
 category = "3D Rendering"
 wasm = true
 
+[[example]]
+name = "orthographic_zoom"
+path = "examples/3d/orthographic_zoom.rs"
+doc-scrape-examples = true
+
+[package.metadata.example.orthographic_zoom]
+name = "Orthographic Zoom"
+description = "Shows how to zoom and orbit an orthographic projection camera."
+category = "3D Rendering"
```

Comment:
Shouldn't these examples be under the `camera` category?

Reply:
Agreed!